docs/user-guide.md

```diff
@@ -368,6 +369,17 @@ $ kubectl annotate service my-service "kube-router.io/service.scheduler=sh"
 
 #For destination hashing scheduling use:
 $ kubectl annotate service my-service "kube-router.io/service.scheduler=dh"
+
+#For maglev scheduling use:
+$ kubectl annotate service my-service "kube-router.io/service.scheduler=mh"
+
+# The maglev scheduler can be further tuned with additional options.
+#To use the maglev scheduler's fallback option use:
+$ kubectl annotate service my-service "kube-router.io/service.schedFlags=flag-1"
```

Comment:
I'm not sure if you tested this and found it working or not in your cluster, but I noticed that in the code, we have `schedflags` instead of `schedFlags`. For what it's worth, I like your version better, but we'll probably keep it all lower case for consistency with existing annotations.

* https://github.com/cloudnativelabs/kube-router/blob/master/pkg/controllers/proxy/network_services_controller.go#L75
* https://github.com/cloudnativelabs/kube-router/blob/master/pkg/controllers/proxy/network_services_controller.go#L970

While I haven't specifically tested that annotations are case sensitive, I believe that they are. Either way, its probably best to have it all lower case to match the code.

Reply:
You're right. Thankfully, in practice I've used the correct  casing and only misspelled it in this PR.